### PR TITLE
DX: enforce `import`s to top by default

### PR DESCRIPTION
**Pull request by shenvitor**
Temporary fix for #269

### CODE REVIEW

File: pyproject.toml
[the remark this answers is not in the document]
Comment (shenvitor) [reply pyproject.toml:1]: Done

**Comment by shenvitor** [comment]
This is done already seems like?

**Comment by redeboer** [comment]
See https://github.com/astral-sh/ruff/issues/13025

**Comment by redeboer** [comment]
This could be fixed with https://github.com/ComPWA/policy/pull/392, but it's not the most ideal because you have to list all excepted notebooks with `excluded`.

**Review by redeboer — APPROVED** [review]

**Comment by redeboer** [comment]
CI should be fixed again once https://github.com/ComPWA/compwa.github.io/pull/294 is through. @shenvitor could you review that PR?

**Pull request merged**